README.md

```diff
@@ -19,9 +19,10 @@ This project can be cloned and re-configured to fit your needs but we highly sug
 Here's how this example project was created:
 
 1. Create a new project directory: `$ mkdir example.com && cd example.com`
-2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git && rm -rf trellis/.git`
+2. Clone Trellis: `$ git clone --depth=1 git@github.com:roots/trellis.git trellis && rm -rf trellis/.git`
 3. Clone Bedrock: `$ git clone --depth=1 git@github.com:roots/bedrock.git site && rm -rf site/.git`
 4. Clone Sage: `$ git clone --depth=1 git@github.com:roots/sage.git site/web/app/themes/sage && rm -rf site/web/app/themes/sage/.git`
+5. From Bedrock directory, run `composer install`
```

Comment:
First change is good but this one isn't necessary. Trellis automatically runs `composer install` for you during development.